### PR TITLE
Add contribute.json file.

### PR DESCRIPTION
**Pull request by Osmose**
Add a contribute.json file a la http://contribute.paas.allizom.org/.

### CODE REVIEW

**Comment by Osmose** [comment]
@glogiotatidis r?

File: contribute.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "repository": {
+        "url": "https://github.com/mozilla/sugardough/",
+        "license": "MPL 2.0",
+        "tests": "https://travis-ci.org/mozilla/sugardough"
+    },
+    "participate": {
+        "home": "https://github.com/mozilla/sugardough/",
+        "docs": "https://github.com/mozilla/sugardough/#cooking-sugardough",
+        "irc": "irc://irc.mozilla.org/#webdev",
+        "irc-contacts": ["giorgos", "pmac"]
```
Comment (peterbe) [inline contribute.json]: I wouldn't mind seeing "Osmose" and "peterbe" in there :)

File: contribute.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "sugardough",
+    "description": "A Cookiecutter template for a web application based on Django.",
+    "repository": {
+        "url": "https://github.com/mozilla/sugardough/",
+        "license": "MPL 2.0",
```
Comment (peterbe) [inline contribute.json]: [Watch out](https://github.com/mozilla/sugardough/pull/50)

**Comment by Osmose** [comment]
Updated.

**Comment by glogiotatidis** [comment]
Looks good but let's wait for #50 to get resolved first.

**Comment by glogiotatidis** [comment]
BTW do you want to add contribute.json to the template?

**Comment by peterbe** [comment]
> BTW do you want to add contribute.json to the template?

Yes!! Why didn't we think of this before?! :)

**Comment by Osmose** [comment]
> BTW do you want to add contribute.json to the template?

Yeah, I figured that'd be a separate PR from this one.

**Comment by glogiotatidis** [comment]
looks good r+. Thanks @Osmose :cake:

**Pull request merged**